task/run-script-oci-ta/0.1/README.md

```diff
@@ -0,0 +1,36 @@
+- Generate labels/annotations with custom code. This is useful when `generate-labels` task can not generate the type of labels/annotations you would like to include on the build.
+
+## Params:
+
+## Params:
+
+| name                      | description                                                                                                                 | default value       |required |
+|---------------------------|-----------------------------------------------------------------------------------------------------------------------------|---------------------|---------|
+| enableSymlinkCheck        | Check symlinks in the repo. If they're pointing outside of the repo, the build will fail.                                   |true                 | false   |
+| ociArtifactsExpireAfter   | Expiration date for the trusted artifacts created in the OCI repository. An empty string means the artifacts do not expire. | ""                  | false   |
+| ociStorage                | The OCI repository where the Trusted Artifacts are stored.                                                                  |                     | true    |
+| SCRIPT_IMAGE              | The image to run the script in                                                                                              |                     | true    |
+| SCRIPT                    | The script to launch                                                                                                        |                     | true    |
+| SCRIPT_WORKDIR            | The directory to launch the script from                                                                                     | /var/workdir/script | false   |
+| SCRIPT_ARTIFACT_DIRECTORY | The directory to include on the output Trusted Artifact                                                                     | /var/workdir/script | false   |
+| STORAGE_DRIVER            | Storage driver to configure for buildah                                                                                     | vfs                 | false   |
+| HERMETIC                  | Determines if build will be executed without network access.                                                                | false               | false   |
+| caTrustConfigMapName      | The name of the key in the ConfigMap that contains the CA bundle data.                                                      | ca-bundle.crt       | false   |
+
+## Results:
+
+| name                | description                                                                                     |
+|---------------------|-------------------------------------------------------------------------------------------------|
+| SCRIPT_ARTIFACT     | The Trusted Artifact URI pointing to the artifact with the content of SCRIPT_ARTIFACT_DIRECTORY |
```

Comment:
This result is written by the build-trusted-artifacts. [Doc](https://github.com/konflux-ci/build-trusted-artifacts/blob/main/create-oci.sh#L15) says information about the artifact is written. If it is only the URI for now, could you please rename `SCRIPT_ARTIFACT` to `SCRIPT_ARTIFACT_URI` to be more specific?

Reply:
Yes sure. But take in account then integrating on the pipeline will be slightly more complex. The name follows in purpose the same name that `prefetch-dependencies` does for an easy to understand pattern while modifying the pipeline to integrate it with `buildah` task:

```
-        value: $(tasks.prefetch-dependencies.results.SOURCE_ARTIFACT)
+        value: $(tasks.run-script.results.SCRIPT_ARTIFACT)
```